packages/tasks/src/model-libraries.ts

```diff
@@ -353,6 +353,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"sbs"`,
 	},
+	"geometry-crafter": {
+		prettyLabel: "GeometryCrafter",
+		repoName: "GeometryCrafter",
+		repoUrl: "https://github.com/TencentARC/GeometryCrafter",
+		countDownloads: `path_filename:"diffusion_pytorch_model"`,
```

Comment:
If a user downloads the full repo, this rule would count it as 3 downloads. To avoid duplicated count, we should only count in one for the folders. What do you think of

```suggestion
		countDownloads: `path:"unet_diff/diffusion_pytorch_model.safetensors"`,
```

for instance?

Reply:
Thanks for your reply, i'll fix it soon by only counting the vae weights downloading, without which the unet part is useless.

Reply:
@Wauplin Hi, wauplinm, I've just updated my commit, could you review it again? Thanks.